Formula/go-jira.rb

```diff
@@ -0,0 +1,31 @@
+class GoJira < Formula
+  desc "Simple jira command-line client in Go"
+  homepage "https://github.com/Netflix-Skunkworks/go-jira"
+  url "https://github.com/Netflix-Skunkworks/go-jira/archive/v1.0.12.tar.gz"
+  sha256 "7ce5d6dcc5021f2019b0107c4ad342979d18bb110188b810017b2d94ff384a33"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/gopkg.in/Netflix-Skunkworks/go-jira.v1"
+    dir.install buildpath.children
+
+    ldflags = ["-w", "-s"]
+    cd dir do
+      system "go", "build", "-o", bin/"jira", "-ldflags", ldflags.join(" "), "cmd/jira/main.go"
+    end
+  end
+
+  test do
+    system "#{bin}/jira", "export-templates"
+    template_dir = Dir.home + "/.jira.d/templates/"
```

Comment:
`Dir.home` is `testpath` here. Or it should be, unless sandbox violations for days are what you're trying to achieve 😄. Just use `testpath`, essentially.

Reply:
The files are written to `~/.jira.d/templates`. Neither ruby nor go are strong languages for me though, so I was just kind of brute forcing my way through. Thanks for the clarification

Reply:
There's a sandbox there for `test do`, as well as the build itself, so if it's not simply obeying the `HOME` env you'll find out soon enough heh.